README.md

```diff
@@ -0,0 +1,140 @@
+[![Build Status](https://travis-ci.org/chembl/ChEMBL_Structure_Pipeline.svg?branch=master)](https://travis-ci.org/chembl/ChEMBL_Structure_Pipeline)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+# ChEMBL Structure Pipeline
+
+ChEMBL protocols used to standardise and salt strip molecules. First used in ChEMBL 26.
+
+Check the [wiki](https://github.com/chembl/ChEMBL_Structure_Pipeline/wiki) for some implementation details.
+
+## Installation (it requires RDKit to work)
+
+From source:
+
+    git clone https://github.com/chembl/ChEMBL_Structure_Pipeline.git
+    pip install ./ChEMBL_Structure_Pipeline
+
+From conda:
+
+```bash
+conda install -c chembl structurepipeline
```

Comment:
if possible, I would call the conda package `chemblstructurepipeline` (or `chembl_structure_pipeline`, which reads better). It's not hard to imagine having a name clash with another package if you just use `structurepipeline`

Reply:
makes sense, but then keeping different names for the conda package and the package itself?

Reply:
I don't understand the question.